manifest.yml.j2

```diff
@@ -64,7 +64,9 @@
   'notify-delivery-worker-letters': {'memory': '2G'},
   'notify-delivery-worker-retry-tasks': {},
   'notify-delivery-worker-internal': {},
-  'notify-delivery-worker-broadcasts': {},
+  'notify-delivery-worker-broadcasts': {
+      'CELERYD_PREFETCH_MULTIPLIER': 1,
```

Comment:
here, should it be `CELERYD` or `CELERY`?

Reply:
I think it's `CELERYD`, as we've got it [here](https://github.com/alphagov/notifications-api/blob/master/manifest.yml.j2#L60). (It's also listed [here](https://docs.celeryproject.org/en/stable/userguide/configuration.html#id13).)

Reply:
ahh, cool, thanks! :)